src/server/rdb_load.cc

```diff
@@ -1958,6 +2002,13 @@ auto RdbLoaderBase::ReadStreams() -> io::Result<OpaqueObj> {
 
       SET_OR_UNEXPECT(FetchInt<int64_t>(), consumer.seen_time);
 
+      if (rdbtype >= RDB_TYPE_STREAM_LISTPACKS_3) {
+        SET_OR_UNEXPECT(FetchInt<int64_t>(), consumer.active_time);
+      } else {
+        /* That's the best estimate we got */
```

Comment:
does it also require improvement  or is it aligned with valkey 7.2?

Reply:
It's a copy of Redis code and it's ok